### DISCUSSION TITLE
understanding the output and why some parameters are not listed

### DISCUSSION THREAD

**Discussion by lt72884** [OpenFAST/openfast#1633] [Q&A]
I added some extra parameters to a .dat file and it shows them as requested outputs in the list, but they do not show any output in the actual .sum file. specifically col 7,8,9 that i added

Col Parameter Units

0 Time (s)
1 BldPitch2 (deg)
2 IPDefl1 (m)
3 IPDefl2 (m)
4 TwstDefl1 (deg)
5 TwstDefl2 (deg)
6 TwstDefl3 (deg)
7 RootFzb1 (kN)
8 RootMIP1 (kN-m)
9 RootMzb1 (kN-m)
10 RootMxb2 (kN-m)
11 RootMyb2 (kN-m)
12 RootMzb2 (kN-m)
13 LSShftFys (kN)
14 LSShftFzs (kN)
15 LSSTipMys (kN-m)
16 LSSTipMzs (kN-m)
17 YawBrTDxp (m)
18 YawBrTDyp (m)
19 YawBrMxn (kN-m)
20 YawBrMyn (kN-m)
21 YawBrMzn (kN-m)

Interpolated blade 1 properties:

Node BlFract RNodes DRNodes PitchAxis StrcTwst BMassDen FlpStff EdgStff
(-) (-) (m) (m) (-) (deg) (kg/m) (Nm^2) (Nm^2)
1 0.033 2.858 2.217 0.258 11.100 180.854 1.112E+09 1.140E+09
2 0.100 5.075 2.217 0.296 11.100 183.682 7.967E+08 9.759E+08
3 0.167 7.292 2.217 0.324 11.100 186.510 4.817E+08 8.121E+08
4 0.233 9.508 2.217 0.338 10.407 184.016 2.571E+08 6.655E+08
5 0.300 11.725 2.217 0.332 8.380 171.286 2.067E+08 5.519E+08
6 0.367 13.942 2.217 0.324 6.353 158.557 1.562E+08 4.382E+08
7 0.433 16.158 2.217 0.315 4.327 145.827 1.058E+08 3.246E+08
8 0.500 18.375 2.217 0.308 2.850 130.509 6.886E+07 2.369E+08
9 0.567 20.592 2.217 0.301 2.217 111.223 5.272E+07 1.887E+08
10 0.633 22.808 2.217 0.295 1.583 91.935 3.659E+07 1.406E+08
11 0.700 25.025 2.217 0.286 0.950 72.647 2.045E+07 9.245E+07
12 0.767 27.242 2.217 0.278 0.532 56.249 1.025E+07 5.928E+07
13 0.833 29.458 2.217 0.272 0.380 43.422 7.387E+06 4.459E+07
14 0.900 31.675 2.217 0.265 0.228 30.595 4.524E+06 2.990E+07
15 0.967 33.892 2.217 0.256 0.076 17.767 1.661E+06 1.522E+07

thanks

**Comment by jjonkman** [comment]
Dear @lt72884,

It looks like you asked the same question in a separate discussion topic; I've now responded there: https://github.com/OpenFAST/openfast/discussions/1627 and will close this duplicate discussion.

Best regards,
